src/NzbDrone.Core/ImportLists/Custom/CustomImport.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections.Generic;
+
+using FluentValidation.Results;
+
+using NLog;
+
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Parser;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.ImportLists.Custom
+{
+    public class CustomImport : ImportListBase<CustomSettings>
+    {
+        private readonly ICustomProxy _customProxy;
```

Comment:
I missed this, but this also needed to be renamed when you renamed the interface...